uftrace.c

```diff
@@ -142,6 +143,7 @@ static struct argp_option ftrace_options[] = {
 	{ "time-range", 'r', "TIME~TIME", 0, "Show output within the TIME(timestamp or elapsed time) range only" },
 	{ "patch", 'P', "FUNC", 0, "Apply dynamic patching for FUNCs" },
 	{ "event", 'E', "EVENT", 0, "Enable EVENT to save more information" },
+	{ "recv-cmd", OPT_recv_cmd, "CMDLINE", 0, "Command line that want to execute after tracing data received" },
```

Comment:
What about changing name to "run-cmd" instead?  This is uftrace recv command, so repeating "recv" here seems unnatural.  Also help message can omit "that want" part IMHO.